pkg/cmd/pipeline/logs.go

```diff
@@ -53,15 +53,25 @@ func logCommand(p cli.Params) *cobra.Command {
 	}
 
 	eg := `
-  # show the logs of the latest pipelinerun of given pipeline
-	tkn pipeline logs foo -n bar
+  # show logs interactively for no inputs
+    tkn pipeline logs -n namespace
 
+  # show logs interactively for given pipeline
+    tkn pipeline logs pipeline_name -n namespace
+
+  # show logs of given pipeline for last run
+    tkn pipeline logs pipeline_name -n namespace --last
```

Comment:
I prefer `pipeline` to `pipeline_name` as a pipeline is known by its name - right?